packages/manager/modules/vps/src/vps/constants.js

```diff
@@ -64,11 +64,33 @@ export const RENEW_URL = {
     'https://us.ovhcloud.com/cgi-bin/order/renew.cgi?domainChooser={serviceName}',
 };
 
+export const VPS = {
+  OPTIONS: {
+    AUTOMATED_BACKUP: 'automatedBackup',
+  },
+  OPTION_STATES: {
+    SUBSCRIBED: 'subscribed',
+    RELEASED: 'released',
+  },
+  VERSIONS: {
+    V1_2013: '2013v1',
+    V1_2014: '2014v1',
+    V1_2015: '2015v1',
+    V1_2017: '2017v1',
+    V2_2017: '2017v2',
+    V3_2017: '2017v3',
+    V1_2018: '2018v1',
+    V2_2018: '2018v2',
+    V1_2019: '2019v1',
```

Comment:
Do we need all of these ?

Reply:
No, i juste copy-past from api all available VPS versions.
Should i remove versions that i don't need them ?

Reply:
@JeremyDec , i searched the /vps.json but not found it.

Reply:
Sorry, I meant to look into the following route, https://api.ovh.com/console/#/vps.json